crates/plugin-kv-store/src/test.rs

```diff
@@ -0,0 +1,196 @@
+#[cfg(test)]
+mod tests {
+
+    use crate::{
+        migration::create_plugin_kv_table,
+        store::{PluginKVScope, PluginKeyValueStore},
+    };
+    use lldap_key_value_store::api::store::KeyValueStore;
+    use sea_orm::{Database, DatabaseConnection, DbErr, TransactionTrait};
+
+    pub struct DbConn {
+        pub fileid: &'static str,
+        pub connection: DatabaseConnection,
+    }
+
+    impl Drop for DbConn {
+        fn drop(&mut self) {
+            let filename: String = "kvtests".to_owned() + self.fileid + ".db";
+            let _ = std::fs::remove_file(filename);
+        }
+    }
+
+    async fn prepare_dbconn(id: &'static str) -> Result<DatabaseConnection, DbErr> {
+        let database_url: String = "sqlite://kvtests".to_owned() + id + ".db?mode=rwc";
```

Comment:
You can have an in-memory sqlite DB.

Reply:
Oh, I wasn't aware of that. That also helps me with some tests, where I've created an InMemoryKeyValueStore :joy:

Reply:
Done